packages/contract-addresses/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as _ from 'lodash';
+
+export interface ContractAddresses {
+    erc20Proxy: string;
+    erc721Proxy: string;
+    zrxToken: string;
+    etherToken: string;
+    exchange: string;
+    assetProxyOwner: string;
+    forwarder: string;
+    orderValidator: string;
+}
+
+export enum NetworkId {
+    Mainnet = 1,
+    Ropsten = 3,
+    Kovan = 42,
+}
+
+const networkToAddresses: { [networkId: number]: ContractAddresses } = {
```

Comment:
Rather then `{ [networkId: number]: ContractAddresses }`, let's use `ObjectMap<ContractAddresses>`

Reply:
Here, `networkId` is a `number` so it's not the same thing as `ObjectMap<ContractAddresses>`. I think we should keep it as a `number` because that is how it appears in the constructors in `@0xproject/contract-wrappers` and in a few other places.